luigi/worker.py

```diff
@@ -1127,8 +1132,16 @@ def _keep_alive(self, get_work_response):
             return get_work_response.n_pending_last_scheduled > 0
         elif self._config.count_uniques:
             return get_work_response.n_unique_pending > 0
+        elif get_work_response.n_pending_tasks <= 0:
```

Comment:
how can n_pending_task ever be less than 0?

Reply:
Good question. I guess I didn't pay enough attention to the context.